app/assets/javascripts/discourse/app/components/composer-body.js

```diff
@@ -199,6 +199,34 @@ export default Component.extend(KeyEnterEscape, {
     );
   },
 
+  updateHeightOnViewportResize() {
+    window.addEventListener(
```

Comment:
We should remove this listener at some point to avoid memory leaks. Maybe it can be removed once the composer is dismissed? (In components we have the `willDestroyElement` hook, a little trickier with controllers.)